docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/amazon-aws.md

```diff
@@ -196,37 +196,37 @@ chmod 400 ~/.ssh/ec2-strapi-key-pair.pem
 #### 2. Log in to your server as the default `ubuntu` user:
 
 ::: tip
-In the future, each time you log into your `EC2` server, you will need to add the path to the .pem file, e.g. `ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@12.123.123.11`.
+In the future, each time you log into your `EC2` server, you will need to add the path to the .pem file and add the IP address for your EC2 instance at the end, e.g. `ssh -i ~/.ssh/ec2-strapi-key-pair.pem ubuntu@34.182.83.134`.
```

Comment:
Not to be picky but that's an actual google ip address maybe we should just change it to 1.2.3.4 or something